### PR TITLE
Bug 1739961 - Provide export option from compare view

### PR DESCRIPTION
**Pull request by sorintoma**

### CODE REVIEW

File: ui/css/perf.css [hunk 1]
```diff
@@ -581,3 +581,22 @@ li.pagination-active.active > button {
   color: #dc3545;
   margin-left: 5px;
 }
+
+.download-json-pagionation-container {
```
Comment (esanuandra) [inline ui/css/perf.css:585]: typo for pagination

File: ui/perfherder/compare/CompareTableControls.jsx [hunk 2]
```diff
@@ -300,20 +317,36 @@ export default class CompareTableControls extends React.Component {
...
+                    updateParams={validated.updateParams}
+                    currentPage={page}
+                    count={countPages}
+                  />
+                </Row>
+              )
+            : null}
+          <div>
+            <Button
+              className="btn donwnload-button"
```
Comment (esanuandra) [inline ui/perfherder/compare/CompareTableControls.jsx:336]: type download

File: ui/perfherder/compare/CompareTableView.jsx [hunk 3]
```diff
@@ -192,6 +192,7 @@ export default class CompareTableView extends React.Component {
       ...this.state,
       ...{ tableNames, rowNames },
     });
+
```
Comment (beatrice-acasandrei) [inline ui/perfherder/compare/CompareTableView.jsx:195]: Try to not add random spaces if you don't modify the logic

File: ui/perfherder/compare/CompareView.jsx [hunk 4]
```diff
@@ -216,6 +216,7 @@ class CompareView extends React.PureComponent {
     });
 
     compareResults = new Map([...compareResults.entries()].sort());
+
```
Comment (beatrice-acasandrei) [inline ui/perfherder/compare/CompareView.jsx:219]: Try to not add random spaces if you don't modify the logic

**Review by esanuandra — APPROVED** [review]

File: ui/css/perf.css [hunk 5]
```diff
@@ -581,3 +581,22 @@ li.pagination-active.active > button {
   color: #dc3545;
   margin-left: 5px;
 }
+
+.download-json-pagination-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.download-json-pagination-container .donwnload-button {
```
Comment (beatrice-acasandrei) [inline ui/css/perf.css:590]: Typo "donwnload"

File: ui/css/perf.css [hunk 6]
```diff
@@ -581,3 +581,22 @@ li.pagination-active.active > button {
...
+  background-color: transparent !important;
+  color: #1f7d8e !important;
+}
+
+.download-json-pagination-container .donwnload-button:hover {
```
Comment (beatrice-acasandrei) [inline ui/css/perf.css:595]: Typo "donwnload"

**Review by beatrice-acasandrei — APPROVED** [review]

**Comment by beatrice-acasandrei** [comment]
In some cases we don't get any results, I think that the button should not be displayed in this cases since there is nothing to download.

**Pull request merged**